tests/netebpfext_unit/netebpfext_unit.cpp

```diff
@@ -1,15 +1,20 @@
 // Copyright (c) Microsoft Corporation
 // SPDX-License-Identifier: MIT
 
-#define CATCH_CONFIG_MAIN
+#include <map>
 
+#define CATCH_CONFIG_MAIN
 #include "catch_wrapper.hpp"
+#include "netebpf_ext_helper.h"
 
-#include "ebpf_registry_helper.h"
-
-#include "net_ebpf_ext.h"
-
-ebpf_registry_key_t ebpf_root_registry_key = HKEY_CURRENT_USER;
-void** _net_ebpf_ext_driver_device_object;
+TEST_CASE("start_stop_test", "[netebpfext]")
+{
+    netebpf_ext_helper_t helper;
 
-TEST_CASE("empty_test", "[netebpfext]") {}
\ No newline at end of file
+    // Get list of program info providers (attach points and helper functions.
+    auto guids = helper.program_info_provider_guids();
```

Comment:
general rule: don't use auto unless the type is obvious from the right side of the `=`